docs/source/chaincode_lifecycle.md

```diff
@@ -94,8 +94,7 @@ automatically create a file in this format.
 
 *The chaincode is packaged separately by Org1 and Org2. Both organizations use
 MYCC_1 as their package label in order to identify the package using the name
-and version. It is not necessary for organizations to use the same package
```

Comment:
Different orgs can indeed use different labels, and even different chaincodes. An organization associates a chaincode package with a chaincode when they do approveformyorg lifecycle step.

Reply:
Cool! it seems packageID linking up with chaincodeID is established during approveformyorg
I will revert this line
Thanks for your clarification, Dave,